### ISSUE TITLE
install the adapter, but what exactly I have to do

### ISSUE THREAD

**Issue by radebold** [iobroker-community-adapters/ioBroker.sourceanalytix#266]
So i have installed the adapter, but what now ?
It's flagged green, but in objects I can't see anything.

I need additional scripts ? I tried to read in diff sites, but can't find more instructions

Please help.

**Comment by raintonr** [comment]
Yes, it isn't very clear.

Once this adapter is activated, visit the admin interface object tree. You will see a little settings icon (wrench) to the right of each state. Click this and you are able to enable analysis for that state.

**Comment by radebold** [comment]
Oh yes, now I understood

Robin Rainton <notifications@github.com> schrieb am Mo. 14. Dez. 2020 um
17:19:

**Comment by radebold** [comment]
Hi Robin,

it is possible to maintenance old data also ?
So I have installed the adapter last week and it would be great to put in
the data for the whole year.

Thx.
Thomas

Am Di., 15. Dez. 2020 um 20:18 Uhr schrieb Thomas Radebold <
radebold@gmail.com>:

**Comment by raintonr** [comment]
> it is possible to maintenance old data also ? So I have installed the adapter last week and it would be great to put in the data for the whole year.

Noting that I am not the author of this adapter so not sure how it is supposed to work...

Do you have data for the whole year stashed away somewhere? I believe there is a config option for each source where you can put in a starting value. Ie. you have a kWh meter that you manually read - I think you can put that manual reading into the config so that this adapter starts from that value instead of zero.

**Comment by radebold** [comment]
yes, i have a database with all energy data since 5 years ago (all 5
minutes :-) )

I have tried to put this data into the objects, but it was set again to 0.
I think it takes from the history table, but thats is nearly empty,
depending on the start last week

Am Mi., 16. Dez. 2020 um 09:54 Uhr schrieb Robin Rainton <
notifications@github.com>:

**Comment by raintonr** [comment]
> yes, i have a database with all energy data since 5 years ago (all 5 minutes :-) )

Sounds to me that would be best kept within InfluxDB or similar.

**Comment by DutchmanNL** [comment]
i will ensure documentation before year end, needed already long ago sorry guys

**Comment by radebold** [comment]
Hi Dutchman,

my iobroker will not update with the standard process to the alpha3 version.
How i can handle that ?

Thx.
Thomas

Am Mi., 16. Dez. 2020 um 10:51 Uhr schrieb Dutchman <
notifications@github.com>:

**Comment by DutchmanNL** [comment]
Please install from git, i will adres the semver issue in our next dev-meeting

**Comment by DutchmanNL** [comment]
The documentation was rewritten for [v0.5.1](https://github.com/DrozmotiX/ioBroker.sourceanalytix/releases/tag/v0.5.1).

The [README](https://github.com/DrozmotiX/ioBroker.sourceanalytix/blob/main/README.md) now has a full walkthrough with Admin screenshots:

1. **Configure the adapter instance** — which statistics get created.
2. **Create price definitions** — the categories your source states will use.
3. **Activate a source state** — via the wrench icon on the state in **Objects**, not through the adapter instance.

Step 3 is the part that was missing here: SourceAnalytix does not show anything until you enable it on a specific source state. No extra scripts are needed. There is also a *Troubleshooting* section covering the common reasons a source is not initialized.

If there is no feedback by **2026-08-11**, I will close this as resolved. It can be reopened at any time.
